logging/cluster-logging-viewing.adoc

```diff
@@ -1,11 +1,16 @@
 :context: cluster-logging-viewing
 [id="cluster-logging-viewing"]
-= Viewing cluster logs
+= Viewing pod logs for a resource
```

Comment:
Is this a duplicate of: https://github.com/openshift/openshift-docs/pull/30366/files#diff-96af6da46e87b1f90ac15098b10c8fa60da32ea69046a65c52b2cf17c294d5a6R6 

Do we need both sections? Can we make it one module and use it in multiple places? Does it need to be in 2 places?